tests-arquillian/src/test/java/org/jboss/weld/tests/ejb/mdb/EJBTest.java

```diff
@@ -51,20 +48,16 @@
                 .addAsModule(
                         ShrinkWrap.create(BeanArchive.class, Utils.getDeploymentNameAsHash(EJBTest.class))
                                 .addPackage(EJBTest.class.getPackage())
-                        //.addAsManifestResource(EJBTest.class.getPackage(), "test-destinations-service.xml", "test-destinations-service.xml")
                 );
     }
 
-    @Category(Broken.class)
     @Test
-    @Ignore
-    // TODO Need a way to deploy test-destinations-service.xml to JBoss AS
     public void testMdbUsable(Control control) throws Exception {
         InitialContext ctx = new InitialContext();
         QueueConnectionFactory factory = (QueueConnectionFactory) ctx.lookup("ConnectionFactory");
         QueueConnection connection = factory.createQueueConnection();
         QueueSession session = connection.createQueueSession(false, Session.AUTO_ACKNOWLEDGE);
-        Queue queue = (Queue) ctx.lookup("queue/testQueue");
+        Queue queue = (Queue) ctx.lookup("java:jboss/exported/jms/queue/test");
```

Comment:
Is it safe to use a hard-coded JNDI name? I mean the name of the test queue could change in the next WildFly version and so might not be able to run the tests against multiple WildFly versions...